README.md

```diff
@@ -0,0 +1,110 @@
+# Chivalry 2 Unchained Launcher (C2UL)
+
+- [What is the Chivalry 2 Unchained Launcher (C2UL)?](#what-is-the-chivalry-2-unchained-launcher-c2ul)
+- [Features](#features)
+- [Join the Community](#community)
+- [Installation](#installation)
+  - [Installation Instructions](#installation-instructions)
+- [General Use Instructions](#general-use-instructions)
+  - [How to Use the Tool](#how-to-use-the-tool)
+  - [Troubleshooting](#troubleshooting)
+  - [Note](#note)
+- [Developer-Focused App Structure and Details](#developer-focused-app-structure-and-details)
+- [Assets](#assets)
+- [License](#license)
+- [Contributing](#contributing)
+
+## What is the Chivalry 2 Unchained Launcher (C2UL)?
+
+The Chivalry 2 Unchained Launcher (C2UL) is a graphical user interface (GUI) tool designed to enhance the Chivalry 2 gaming experience. By providing an easy way to launch the game, manage mods, and customize settings, C2UL expands the capabilities of Chivalry 2, fostering community growth and enriching the overall player experience.
+
+## Features
+
+- **Launch Chivalry 2**: Easily launch Chivalry 2 with custom configurations.
+- **Mod Management**: Download and manage mods for Chivalry 2.
+- **Customization**: Customize the launcher with various settings and preferences.
```

Comment:
I think we can drop this feature, since the settings stuff is just a necessary evil more than a feature I think

Reply:
It might be expanded later and could be considered a stub.